Assets/MRTK/SDK/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -115,6 +115,8 @@ public override void OnPreSceneQuery()
         {
             using (OnPreSceneQueryPerfMarker.Auto())
             {
+                if (!IsInteractionEnabled) return;
```

Comment:
Hmm yeah, that's a good callout! Are you referring to the debug rays in the editor or the actual visible rays in-app?
If the second, I'm unsure if I agree. The FocusProvider doesn't do any actual raycasting if `IsInteractionEnabled` is false

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/2338a9f02a404e267407553b36e41af8326b7a2e/Assets/MRTK/Services/InputSystem/FocusProvider.cs#L1020-L1027

which means I'm unsure that any visual representation in-app will be properly meaningful.

If the first though, I'd be happy to change this to 
```suggestion
                if (!IsInteractionEnabled && !Application.isEditor) return;
```
or something

Reply:
I think semantically, LinePointer refers to the ability to "point a line in a direction driven by a pose input" but doesn't make any supposition about it's ability to interact with things. Based on what it's subclassing, I can see how it might be intended to be used as an interaction point, so I'm a bit conflicted here. 

That said, I see in OnPostScene Query we just stop doing things if `!IsInteractionEnabled`, so I think I'm fine with the current implementation in the end :D